setup.py

```diff
@@ -0,0 +1,65 @@
+import os.path
+from setuptools import find_packages, setup
+
+# Package data
+# ------------
+_author       = 'covjson'
+_authorEmail  = 'resc@reading.ac.uk'
+_classifiers  = [
+    'Environment :: Console',
+    'Intended Audience :: Developers',
+    'Intended Audience :: Information Technology',
+    'Intended Audience :: Science/Research',
+    'Topic :: Scientific/Engineering',
+    'Development Status :: 4 - Beta',
+    'License :: Copyright :: University of Reading',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python',
+    'Topic :: Internet :: WWW/HTTP',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+]
+_description  = 'Create CovJSON files from common scientific data formats'
+_downloadURL  = 'http://pypi.python.org/pypi/pycovjson/'
+_requirements = ["", "",""]
+_keywords     = ['dataset', 'coverage', 'covjson']
+_license      = 'Copyright :: University of Reading'
+_long_description    = 'A python utility library for creating CovJSON files from common scientific data formats'
+_name         = 'pycovjson'
+_namespaces   = []
+_testSuite    = 'pycovjson.test'
+_url          = 'https://github.com/Reading-eScience-Centre/pycovjson'
+_version      = '1.0.0'
+_zipSafe      = True
+
+# Setup Metadata
+# --------------
+
+def _read(*rnames):
+    return open(os.path.join(os.path.dirname(__file__), *rnames)).read()
+
+_header = '*' * len(_name) + '\n' + _name + '\n' + '*' * len(_name)
+_longDescription = '\n\n'.join([
+    _header,
+    _read('README.md')
+])
+open('doc.txt', 'w').write(_longDescription)
```

Comment:
It's an indication of what the landing page on Pypi would look like. It serves (for me anyway) no other purpose.